### PR TITLE
Updated review screens

### PR DESCRIPTION
**Pull request by Nat-A-M**
Assessment and teaching expertise review screens, with updated routes
Screenshot 2022-12-20 at 16 00 26
Screenshot 2022-12-20 at 15 59 11

### CODE REVIEW

File: app/views/_includes/summary-cards/assessment-expertise.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
           href: "/application/assessment-expertise/add-details-designing"
         }]
       }
-    } if data.anyAssessmentExpertise == "Yes",
+    } if data.isDesigning,
       {
       key: {
         text: "Provide details of your experience in reviewing assessment performance"
```
Comment (dalcJ) [inline app/views/_includes/summary-cards/assessment-expertise.html:67]: Need to add the if data.isX to this row aswell.

File: app/views/_includes/summary-cards/teaching-expertise.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <ul class="govuk-list">
```
Comment (dalcJ) [inline app/views/_includes/summary-cards/teaching-expertise.html:3]: this whole <ul> chunk can be deleted

File: app/views/_includes/summary-cards/teaching-expertise.html [hunk 3]
```diff
@@ -18,63 +18,48 @@
...
-      },
-      actions: {
-        items: [{
-          text: "Change",
-          visuallyHiddenText: "Provide details of your teaching, lecturing or training expertise",
-          href: "/application/teaching-expertise/add-details"
-        }]
-      }
-    } if data.anyTeachingExpertise == "Yes"
+    }
```
Comment (dalcJ) [inline app/views/_includes/summary-cards/teaching-expertise.html:63]: add `if data.isTraining` to the end of this line. So this row is conditional on if they have selected that type of expertise.

File: app/routes.js [hunk 4]
```diff
@@ -189,11 +189,33 @@ router.get('/assessment-type-answer', function (req, res) {
   let assessmentMarkingRegex = new RegExp(/Marking assessments/i)
   let assessmentModeratingRegex = new RegExp(/Moderating assessments/i)
   let assessmentDesigningRegex = new RegExp(/Designing and setting assessments/i)
+  let assessmentSeniorRegex = new RegExp(/Designing and setting assessments/i)
```
Comment (dalcJ) [inline app/routes.js:192]: this one should be `let assessmentReviewingRegex = new RegExp(/Reviewing assessment performance/i)`

File: app/routes.js [hunk 5]
```diff
@@ -189,11 +189,33 @@ router.get('/assessment-type-answer', function (req, res) {
...
+    isModerating = false
+  }
+
+  if (isDesigning == true) {
+    // this is to use in the nunjucks view
+    req.session.data.isDesigning = true 
+  } else {
+    isDesigning = false
+  }
+
```
Comment (dalcJ) [inline app/routes.js:218]: Need to add the one for Reviewing 

`if (isReviewing == true) {
    // this is to use in the nunjucks view
    req.session.data.isReviewing = true 
  } else {
    isReviewing = false
  }`

File: app/routes.js [hunk 6]
```diff
@@ -189,11 +189,33 @@ router.get('/assessment-type-answer', function (req, res) {
   let assessmentMarkingRegex = new RegExp(/Marking assessments/i)
   let assessmentModeratingRegex = new RegExp(/Moderating assessments/i)
   let assessmentDesigningRegex = new RegExp(/Designing and setting assessments/i)
+  let assessmentSeniorRegex = new RegExp(/Designing and setting assessments/i)
 
   let isMarking = assessmentMarkingRegex.test(assessmentExpertise)
   let isModerating = assessmentModeratingRegex.test(assessmentExpertise)
   let isDesigning = assessmentDesigningRegex.test(assessmentExpertise)
 
+  if (isMarking == true) {
```
Comment (dalcJ) [inline app/routes.js:198]: add above this for Reviewing: 

`let isReviewing = assessmentReviewingRegex.test(assessmentExpertise)`

**Review by dalcJ — CHANGES_REQUESTED** [review]
See comments

File: app/views/_includes/summary-cards/teaching-expertise.html (hunk 3 above)
Comment (Nat-A-M) [reply app/views/_includes/summary-cards/teaching-expertise.html:63]: When I add this line, the q&a does not appear on the review screen. It only appears to work when i remove it. I'm not sure why.

**Review by dalcJ — APPROVED** [review]

**Pull request merged**